### PR TITLE
built-ins/BigInt/*: make all indentation consistent (depth & character)

### PR DESCRIPTION
**Pull request by rwaldron**

### CODE REVIEW

File: test/built-ins/BigInt/asIntN/bigint-tobigint-toprimitive.js [hunk 1]
```diff
@@ -22,64 +22,64 @@ assert.sameValue(BigInt.asIntN(2, {
   },
   valueOf: err,
   toString: err
-}), 1n, "ToPrimitive: @@toPrimitive takes precedence");
+}), 1 n, "ToPrimitive: @@toPrimitive takes precedence");
```
Comment (leobalter) [inline test/built-ins/BigInt/asIntN/bigint-tobigint-toprimitive.js]: this should probably be reported as an issue of jsbeautify.

Comment (ljharb) [reply test/built-ins/BigInt/asIntN/bigint-tobigint-toprimitive.js]: see also https://github.com/tc39/test262/pull/1416#discussion_r167602054

Comment (rwaldron) [reply test/built-ins/BigInt/asIntN/bigint-tobigint-toprimitive.js]: Sorry, I thought I found and fixed all of these!

**Comment by rwaldron** [comment]
Fixed!

**Pull request merged**